From 1d944e17972081c327097f30cf3aab345bac09c5 Mon Sep 17 00:00:00 2001
From: Vlad Pelin <vlad.pelin@nxp.com>
Date: Mon, 11 Oct 2021 17:03:16 +0300
Subject: [PATCH 2/5] pkcs: update .gitingore

added obj folders and .so output to .gitignore

Issue: ALB-7684
Upstream-Status: Pending 

Signed-off-by: Vlad Pelin <vlad.pelin@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 .gitignore | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/.gitignore b/.gitignore
index ae70ed8..0b09525 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,2 +1,4 @@
 examples/pkcs-keyop
-pkcs11-hse.so
+*.so*
+libhse/obj
+libpkcs/obj
-- 
2.17.1

